### PR TITLE
PI-910 fix IE11 compatibility issues

### PR DESCRIPTION
**Pull request by thellimist**

### CODE REVIEW

File: src/js/containers/Header/Header.js [hunk 1]
```diff
@@ -6,6 +6,7 @@ import ActiveZoneSelector from '../../containers/ActiveZoneSelector/ActiveZoneSe
 import { isLoggedIn } from '../../utils/Auth/Auth';
 import { getAbsoluteUrl } from '../../selectors/config';
 import UnderAttackButton from '../../containers/UnderAttackButton/UnderAttackButton';
+import { LayoutColumn } from 'cf-component-layout';
```
Comment (jwineman) [inline src/js/containers/Header/Header.js]: Why are you importing `<LayoutColumn/>` without using it?

File: src/js/utils/utils.js [hunk 2]
```diff
@@ -18,10 +18,14 @@ function beginsWith(needle, haystack){
     return (haystack.substr(0, needle.length) == needle);
 }
 
+function endsWith(str, suffix) {
+    return str.indexOf(suffix, str.length - suffix.length) !== -1;
```
Comment (jwineman) [inline src/js/utils/utils.js]: Why not

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/endsWith

File: src/js/containers/Header/Header.js (hunk 1 above)
Comment (thellimist) [inline src/js/containers/Header/Header.js]: it is being used. It was never imported. I've no idea how it worked before

File: src/js/utils/utils.js (hunk 2 above)
Comment (thellimist) [inline src/js/utils/utils.js]: Because `developer.mozilla` not `developer.ie` :/

File: src/js/containers/Header/Header.js (hunk 1 above)
Comment (jwineman) [inline src/js/containers/Header/Header.js]: lol

**Comment by jwineman** [comment]
LGTM

**Pull request merged**